assets/addon_browser/addons.conf

```diff
@@ -603,4 +603,15 @@
       "github": "https://github.com/Chicken/Auth/tree/master/BlueMap/PrivateMaps"
     }
   }
+  {
+    name: "OpenPaC BlueMap - Remapped"
+    description: "Show land claims from the Open Parties and Claims mod as markers in BlueMap."
+    author: "DreamyLynn"
+    apiVersion: "2"
+    platforms: ["fabric", "quilt"]
```

Comment:
`quilt` is not a valid platform for this config at the moment..
I think it's clear that it supports quilt if fabric itself is already mentioned :)

Reply:
You can also just add an icon for quilt in `/assets/addon_browser/platforms/`

Reply:
added an icon for quilt